cultcargo/builder/cargo-manifest.yml

```diff
@@ -129,3 +129,28 @@ images:
     versions:
       base:
         package: git+https://github.com/ratt-ru/CubiCal@noxcal
+
+  mosaic-queen:
+    assign:
+      pre_install: RUN apt update && apt install -y montage
```

Comment:
Suggest adding ``&& rm -rf /var/lib/apt/lists/* && rm -rf /var/cache/apt/archives`` to reduce the image size.